Keas.Mvc/Handlers/VerifyRoleAccessHandler.cs

```diff
@@ -32,11 +39,16 @@ protected override async Task HandleRequirementAsync(AuthorizationHandlerContext
                 {
                     team = mvcContext.RouteData.Values["teamName"].ToString();
                 }
-            }
+            }  
+            if (string.IsNullOrWhiteSpace(team)) 
+             {
+                var tempData = _tempDataDictionaryFactory.GetTempData(_httpContext.HttpContext);
+                team = tempData["TeamName"] != null ? tempData["TeamName"].ToString() : ""; 
```

Comment:
@jcubbage why not just do team = `string.format("{0}", tempData["TeamName"})` or maybe even simpler use `Convert.ToString(tempData["TeamName"]`.  That'll take care of nulls and formatting as a string